portal-kernel/src/com/liferay/portal/kernel/workflow/comparator/packageinfo

```diff
@@ -1 +1 @@
-version 7.3.0
\ No newline at end of file
+version 8.0.0
```

Comment:
@naraandrade Please extract this file change from this commit and create a new one with the following message: "LPS-123224 Semver". 

PS: changes like this one (version bump) go in a specific commit always